watertap/unit_models/pressure_exchanger.py

```diff
@@ -140,6 +146,16 @@ def build(self):
             units=pyunits.dimensionless,
             doc='Pressure exchanger efficiency')
 
+        if self.config.has_mass_transfer:
+            self.solution_transfer_fraction = Var(
+                self.flowsheet().config.time,
+                self.config.property_package.component_list,
```

Comment:
Do you still need to index by component if this is now just a fraction for the solution?

Reply:
We still want to have an option of index by component incase they can have different transfer fractions (e.g. H2O, TDS, NaCl, etc. ) Is there a better/cleaner way to do that? ( I figured doing it by solute/solvent, is not necessary here).

Reply:
The problem with doing that is that model structure is then dependent on whether you use component indexing or not. In these cases, I would generally go with always indexing by component - if they are all the same then we can just fix the entire indexed variable to the same value (which can be done in a single line of code).

Reply:
Ah,  I thought you decided to not consider different transfer fractions, hence renaming it to `solution_transfer_fraction`, but I see now that you just want to account for all components under one variable name. Maybe renaming it to something like `comp_transfer_fraction` might be clearer.

Reply:
That sounds reasonable I will update the name.

Reply:
The name still needs to be updated. I think a more IDAES name would be `mass_transfer_fraction_comp`

Reply:
Fixed this.